apps/admin-gui/src/app/shared/components/dialogs/add-member-to-resource-dialog/add-member-to-resource-dialog.component.ts

```diff
@@ -34,6 +34,8 @@ export class AddMemberToResourceDialogComponent implements OnInit {
   }
 
   theme: string;
+  loading = false;
+  processing = false;
```

Comment:
If everything is set just to loading and you want select resource it becomes a bit buggy and you are unable to select resource. I think that its because resource-select is calling method (setResource) using api and reload of select breaks something. So I also added processing to disable continue button. Next dialog step works only with processing.